.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -1,49 +1,109 @@
-1. - [ ] Tag SDK version (e.g. "v2.1.300-SDK") with the same annotation, also preferably signed
-1. - [ ] Push these tags to GitHub
-      - `git push <remote> v2.1.0-runtime && git push <remote> v2.1.300-SDK`
-            
-        Do not use "git push --tags" unless you have a fresh repo with no other tags - this will push all your tags.
-1. - [ ] If there's a respin, retag the with the new changes. A sub-patch version tag for SDK and runtime would have to be generated. E.g - v2.1.300.1-SDK
-1. - [ ] Notify the distro maintainer/s about the release tags
-1. - [ ] Download the tarball from CI
-1. - [ ] Upload the tarball to Azure blob feed for source-build
-
-            <Source-build-blob-feed-container>/redhat/<branch_version>/<SDK_version>/dotnet-<sdk_version>-<RID>.tar.gz
-
-1. - [ ] Similarly, download Private.SourceBuilt.Artifacts.XX.tar.gz from CI and upload it to source-built-artifacts blob container
-1. - [ ] Write a post-mortem for the release
-
-### Additional notes
-
-See the .NET Core A&D OneNote -> Tarball section -> Source-build release checklist.
\ No newline at end of file
+1.  - [ ] Create a doc to take ongoing notes on problems with the process, workarounds, and fixes.
+      - This is useful to make sure context is available to review later. It may end up blank if the release goes very smoothly.
+      - The common location is ".NET Core A&D" OneNote -> "Servicing" section -> "{Month} {YYYY}".
+        - There are other notes on servicing in this OneNote. It may be useful to review if something goes wrong to see if it's been fixed before.
+1.  - [ ] [Internal] Ensure internal/release/X.Y branch is up to date with mirrored release/X.Y branch.
+1.  - [ ] Update to new version.
+      - [2.1] [/Documentation/servicing/update-2.1.md](https://github.com/dotnet/source-build/tree/release/3.1/Documentation/servicing/update-2.1.md)
+      - [3.1] [/Documentation/servicing/update-3.1.md](https://github.com/dotnet/source-build/tree/release/3.1/Documentation/servicing/update-3.1.md)
+      - Start a new working branch and PR for WIP so others can easily look if needed and CI starts running.
+        - [Internal] For internal builds, use <https://dev.azure.com/dnceng/internal/_git/dotnet-source-build>, branch name `internal/release/<SDK-version>-<month name><YYYY>`.
+1.  - [ ] Version update cleanup and confirmations
+      - [2.1]
+        - [ ] Verify that `source-build/ProdConFeed.txt` contains the latest feed required for the release.
+          - Build number should match ProdCon build number.
+          - Ensure auth details are **not** included in this file.
+        - [ ] Verify/update the ASP.NET versions (`MicrosoftAspNetCoreAllPackageVersion` and `MicrosoftAspNetCoreAppPackageVersion`) in dependencies.props.
+        - [ ] Verify/update the `OfficialBuildId`s in updated repos (usually coreclr, corefx, core-setup).
+        - [ ] Update the `DotnetCLIVersion.txt` SDK version to N-1.
```

Comment:
`N-1` is the previous bugfix release for this release branch, right?

Reply:
Yeah, I suppose more specifically the Microsoft release, whether or not it's available in *all* distros. I'll add a bit of detail, thanks.